File: appcompat/src/androidTest/java/com/kfaraj/support/widget/SupportEditTextTest.java
package com.kfaraj.support.widget;

import android.content.Context;
import android.support.test.InstrumentationRegistry;
import android.support.test.filters.SmallTest;
import android.support.test.runner.AndroidJUnit4;

import com.kfaraj.support.view.TintableTextViewTest;

import org.junit.Before;
import org.junit.runner.RunWith;

@RunWith(AndroidJUnit4.class)
@SmallTest
public class SupportEditTextTest extends TintableTextViewTest<SupportEditText> {

    @Before
    public void setUp() {
        Context context = InstrumentationRegistry.getContext();
        mTextView = new SupportEditText(context);
    }

}

File: appcompat/src/androidTest/java/com/kfaraj/support/widget/SupportEditTextTest.kt
package com.kfaraj.support.widget

import androidx.test.filters.SmallTest
import androidx.test.platform.app.InstrumentationRegistry
import com.kfaraj.support.view.TintableTextViewTest
import org.junit.Before

@SmallTest
class SupportEditTextTest :
    TintableTextViewTest<SupportEditText>() {

    @Before
    fun setUp() {
        val context = InstrumentationRegistry.getInstrumentation().targetContext
        textView = SupportEditText(context)
    }

}
